crates/sui-framework/packages/move-stdlib/sources/type_name.move

```diff
@@ -2,58 +2,58 @@
 // SPDX-License-Identifier: Apache-2.0
 
 /// Functionality for converting Move types into values. Use with care!
-module std::type_name {
-    use std::ascii::{Self, String};
-    use std::address;
-
-    /// ASCII Character code for the `:` (colon) symbol.
-    const ASCII_COLON: u8 = 58;
-
-    /// ASCII Character code for the `v` (lowercase v) symbol.
-    const ASCII_V: u8 = 118;
-    /// ASCII Character code for the `e` (lowercase e) symbol.
-    const ASCII_E: u8 = 101;
-    /// ASCII Character code for the `c` (lowercase c) symbol.
-    const ASCII_C: u8 = 99;
-    /// ASCII Character code for the `t` (lowercase t) symbol.
-    const ASCII_T: u8 = 116;
-    /// ASCII Character code for the `o` (lowercase o) symbol.
-    const ASCII_O: u8 = 111;
-    /// ASCII Character code for the `r` (lowercase r) symbol.
-    const ASCII_R: u8 = 114;
-
-    /// The type is not from a package/module. It is a primitive type.
-    const ENonModuleType: u64 = 0;
-
-    public struct TypeName has copy, drop, store {
-        /// String representation of the type. All types are represented
-        /// using their source syntax:
-        /// "u8", "u64", "bool", "address", "vector", and so on for primitive types.
-        /// Struct types are represented as fully qualified type names; e.g.
-        /// `00000000000000000000000000000001::string::String` or
-        /// `0000000000000000000000000000000a::module_name1::type_name1<0000000000000000000000000000000a::module_name2::type_name2<u64>>`
-        /// Addresses are hex-encoded lowercase values of length ADDRESS_LENGTH (16, 20, or 32 depending on the Move platform)
-        name: String,
-    }
-
-    /// Return a value representation of the type `T`.  Package IDs
-    /// that appear in fully qualified type names in the output from
-    /// this function are defining IDs (the ID of the package in
-    /// storage that first introduced the type).
-    public native fun get<T>(): TypeName;
-
-    /// Return a value representation of the type `T`.  Package IDs
-    /// that appear in fully qualified type names in the output from
-    /// this function are original IDs (the ID of the first version of
-    /// the package, even if the type in question was introduced in a
-    /// later upgrade).
-    public native fun get_with_original_ids<T>(): TypeName;
-
-    /// Returns true iff the TypeName represents a primitive type, i.e. one of
-    /// u8, u16, u32, u64, u128, u256, bool, address, vector.
-    public fun is_primitive(self: &TypeName): bool {
-        let bytes = self.name.as_bytes();
-        bytes == &b"bool" ||
+module std::type_name;
+
+use std::{address, ascii::{Self, String}};
```

Comment:
not sure I like the `use` grouping here. Strong feelings? If not, maybe revert for this PR

Reply:
It's what formatter does these days.